app/assets/javascripts/leaflet.map.js

```diff
@@ -59,7 +59,7 @@ L.OSM.Map = L.Map.extend({
         }).prop("outerHTML")
       });
 
-      attribution += credit.donate ? " &hearts; " : ". ";
+      attribution += credit.donate ? " ♥️ " : ". ";
```

Comment:
Personal Preference: this punctuation looks a bit out of place.

Unrelated to your PR though, so lets not change it ^^

```suggestion
      attribution += credit.donate ? " ♥️ " : " ";
```